src/shared/Actions.ts

```diff
@@ -20,7 +33,7 @@ export default {
   submit(o = {}) {
     this.$submitting = true;
 
-    const exec = isValid => isValid
+    const exec:any = isValid => isValid
```

Comment:
No `any` please! There `(isValid: boolean) => Function | Promise`,
and yes bug at 60 at `return utils.isPromise(exec) ? ... : ...`;
And!
My promise must call after success validation! No before!

Reply:
Can you clarify how it should be in the code?